index.js

```diff
@@ -257,9 +257,7 @@
 
                 //BACKSPACE
                 case 8:
-                    if (this.query === '') {
-                        this.cleanUp(true);
-                    } else {
+                    if (this.query !== '') {
```

Comment:
Is there a way to make sure that the `delimiter` (the `@`) is still there? I just want to avoid that the key handlers don't get cleaned up if the user removes the `@` as well

Reply:
@ggeles sorry, I forgot to mention you. I am not sure if you received my comment

Reply:
I try to test it on playground it works ok but after few repeats it stops showing dropdown box with the names. I test it on Formly rtf field page

Reply:
@ggeles that was what I was afraid of. I will try to take a look in the afternoon. If I can't find anything to fix that issue I will approve it later tonight or tomorrow. Ok?

Reply:
yes, thank you!